db_stress_tool/db_stress_test_base.cc

```diff
@@ -1707,6 +1742,9 @@ Status StressTest::TestBackupRestore(
       // lock and wait on a background operation (flush).
       create_opts.flush_before_backup = true;
     }
+    create_opts.decrease_background_thread_cpu_priority = thread->rand.OneIn(2);
```

Comment:
Is it possible to log the options if the following operation failed? So we know if a new option causes stress test failure. The same for restore, ingestion and manual compaction below.

Reply:
Yes

Reply:
Fixed

```
.. under specified BackupEngineOptions: share_table_files: 1, share_files_with_checksum: 1, share_files_with_checksum_naming: 2147483650, schema_version: 2, max_background_operations: 1, backup_rate_limiter: 0x7f9ac7c42500, restore_rate_limiter: 0x7f9ac7c42280, CreateBackupOptions: flush_before_backup: 1, decrease_background_thread_cpu_priority: 1, background_thread_cpu_priority: 0, RestoreOptions: keep_log_files: 1 (Empty string or missing field indicates default option or value is used)
```